### PR TITLE
updates

### PR DESCRIPTION
**Pull request by kzidane**
* moved directory structure to `Makefile`, keeping only `bin/` and `share/` in the root of this repo.
* automated deployment to mirror.
* ensured `php -S` is already listening before outputting URL (to avoid opening URL too soon)

### CODE REVIEW

File: bin/phpliteadmin [hunk 1]
```diff
@@ -100,6 +100,9 @@ fi
 php -S 0.0.0.0:$PORT -t "$temp" > /dev/null 2>&1 &
 pid=$!
 
+# wait till server is listening
```
Comment (dmalan) [inline bin/phpliteadmin]: Technically `'til`.

File: bin/phpliteadmin [hunk 2]
```diff
@@ -100,6 +100,9 @@ fi
...
+while curl -I http://localhost:$PORT &> /dev/null; do sleep .5; done
```
Comment (dmalan) [inline bin/phpliteadmin]: What happens if this loops infinitely? Should we have a timeout?

**Review by dmalan — DISMISSED** [review]

File: bin/phpliteadmin (hunk 2 above)
Comment (kzidane) [reply bin/phpliteadmin]: Added per c2906e7!

**Comment by kzidane** [comment]
@dmalan any other takes before we :shipit:?

**Review by dmalan — APPROVED** [review]

**Pull request merged**